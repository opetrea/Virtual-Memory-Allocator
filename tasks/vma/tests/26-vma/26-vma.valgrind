==91927== Memcheck, a memory error detector
==91927== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==91927== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==91927== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==91927== Parent PID: 91926
==91927== 
==91927== 
==91927== HEAP SUMMARY:
==91927==     in use at exit: 0 bytes in 0 blocks
==91927==   total heap usage: 46 allocs, 46 frees, 9,336 bytes allocated
==91927== 
==91927== All heap blocks were freed -- no leaks are possible
==91927== 
==91927== For lists of detected and suppressed errors, rerun with: -s
==91927== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
