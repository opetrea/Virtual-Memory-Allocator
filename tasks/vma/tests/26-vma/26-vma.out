Total memory: 0x3E8 bytes
Free memory: 0x2E4 bytes
Number of allocated blocks: 6
Number of allocated miniblocks: 6

Block 1 begin
Zone: 0x64 - 0x6E
Miniblock 1:		0x64		-		0x6E		| RW-
Block 1 end

Block 2 begin
Zone: 0xC8 - 0xDC
Miniblock 1:		0xC8		-		0xDC		| RW-
Block 2 end

Block 3 begin
Zone: 0x12C - 0x136
Miniblock 1:		0x12C		-		0x136		| RW-
Block 3 end

Block 4 begin
Zone: 0x190 - 0x19A
Miniblock 1:		0x190		-		0x19A		| RW-
Block 4 end

Block 5 begin
Zone: 0x1F4 - 0x1FE
Miniblock 1:		0x1F4		-		0x1FE		| RW-
Block 5 end

Block 6 begin
Zone: 0x2BC - 0x384
Miniblock 1:		0x2BC		-		0x384		| RW-
Block 6 end
